### DISCUSSION TITLE
Extension headers and options section

### DISCUSSION THREAD

**Discussion by EduardVasilenko** [becarpenter/book6#38] [General]
Hi Brian,
I am not sure, but by my taste, it is better to mention that “HbH or DO options have a classification in the high order bits for nodes that may not understand the option: 0) skip option; 1) discard packet; 2) discard any packet with ICMP response; 3) discard unicast packet with ICMP response.”
Eduard

**Comment by becarpenter** [comment]
Will fix

**Comment by becarpenter** [comment]
Closed as done
